src/doc/dev_manual/LiveCustomerResponse.rst

```diff
@@ -178,12 +195,16 @@ the associated GitHub issue in response to the *initial* email, the less likely
 this can occur. In addition, boilerplate guidance in an issue template will help
 to mitigate this by informing ``visit-users`` members of this behavior.
 
-For any work the **Primary** engages in, there should be an associated issue
-for tracking that work.
+For any work the **Primary** performs, even if it is a rather trivial amount
+of work to resolve, there should be an associated issue for tracking that work.
+Tracking even the trivial issues can help identify patterns that we may be able
+to later mine to identify process improvements.
 
 .. danger::
    What about ``visit-help-asc`` email list? Should we handle similarly?
-   Can we just foward to ``visit-users``?  If no to either, what do we do?
+   Can we just forward to ``visit-users``?  If no to either, what do we do?
+
+.. _sre_response_vs_resolution:
 
 Response Time and Response vs. Resolution
 -----------------------------------------
```

Comment:
Can't comment on line 212 because it is too far way from changes, but I just realized that it should be team members', not team member's (apostrophe)